### PR TITLE
Add CI build pipeline

### PR DESCRIPTION
**Pull request by ianschmitz**
Adds a basic smoke test for all 3 platforms by performing a debug build of each.

### CODE REVIEW

**Comment by ianschmitz** [comment]
@kewalker there's a few warnings on each platform. Do any of them look like we can fix easily?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+
+-   Open the **App1** solution in Visual Studio.
+-   Set **App1.UWP** as your startup project.
+-   Change the architecture from _Any CPU_ to _x86_ or _x64_.
+-   Build and run.
 
 ## Documentation
-- Find [further documentation on the quickstart](https://developers.geocortex.com/docs/mobile/quick-start/) on the [Geocortex Development Center](https://developers.geocortex.com/docs/mobile/overview).
+
+-   Find [further documentation on the quickstart](https://developers.geocortex.com/docs/mobile/quick-start/) on the [Geocortex Development Center](https://developers.geocortex.com/docs/mobile/overview).
```
Comment (kewalker) [inline README.md]: The link to the quick-start isn't valid anymore. I think you'll want this one:
https://developers.geocortex.com/docs/mobile/sdk-quick-start

**Review by kewalker — CHANGES_REQUESTED** [review]
Looks great, thanks for adding those checks!
For warnings:
- Android and iOS have warnings around CSharp versions. We may be able to remove those by level-setting the package version explicitly in the projects, and possibly level-setting the version across the different packages.
- Unfortunately I'm not too sure about the mtouch warnings associated with iOS.
- The warning for UWP is for not using a parameter for `OnNavigationFailed`. We could remove the parameter, or remove the method altogether (Mobile project proper doesn't seem to use it).

File: README.md (hunk 1 above)
Comment (ianschmitz) [reply README.md]: Good catch! I fixed in #3.

**Comment by ianschmitz** [comment]
> The warning for UWP is for not using a parameter for `OnNavigationFailed`. We could remove the parameter, or remove the method altogether (Mobile project proper doesn't seem to use it).

I removed the method to get rid of this one.

The others i didn't have any luck overcoming. Looks like there may be some conflicts between Newtonsoft.Json referenced by netstandard2.0 and everything else:

```   
 There was a conflict between "Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" and "Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed".
2>      "Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" was chosen because it was primary and "Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" was not.
2>      References which depend on "Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" [C:\Users\Ian\.nuget\packages\newtonsoft.json\11.0.2\lib\netstandard2.0\Newtonsoft.Json.dll].
2>          C:\Users\Ian\.nuget\packages\newtonsoft.json\11.0.2\lib\netstandard2.0\Newtonsoft.Json.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\newtonsoft.json\11.0.2\lib\netstandard2.0\Newtonsoft.Json.dll".
2>              C:\Users\Ian\.nuget\packages\newtonsoft.json\11.0.2\lib\netstandard2.0\Newtonsoft.Json.dll
2>      References which depend on "Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" [].
2>          C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.API.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.API.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.API.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>          C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Licensing.Client.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Licensing.Client.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Licensing.Client.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>          C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Composition.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Composition.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Composition.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>          C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>          C:\Users\Ian\.nuget\packages\geocortex.workflow.gis\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.GIS.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.workflow.gis\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.GIS.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.gis\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.GIS.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>          C:\Users\Ian\.nuget\packages\geocortex.workflow.runtime\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.Runtime.dll
2>            Project file item includes which caused reference "C:\Users\Ian\.nuget\packages\geocortex.workflow.runtime\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.Runtime.dll".
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.runtime\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.Runtime.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1.Android\bin\Debug\App1.Android.dll
2>              C:\Users\Ian\source\repos\vertigis-mobile-quickstart\App1\App1\bin\Debug\netstandard2.0\App1.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.mobile\5.6.0\lib\MonoAndroid10\Geocortex.Mobile.Android.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.activities.image\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.Activities.Image.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.forms\5.14.2.184\lib\MonoAndroid10\Geocortex.Workflow.Forms.Android.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.forms\5.14.2.184\lib\MonoAndroid10\Geocortex.Workflow.Forms.dll
2>              C:\Users\Ian\.nuget\packages\geocortex.workflow.gis\5.14.2.184\lib\netstandard2.0\Geocortex.Workflow.GIS.dll
```

**Review by kewalker — APPROVED** [review]

**Pull request merged**